viewing_party/party.py

```diff
@@ -0,0 +1,168 @@
+def create_movie(title, genre, rating):
+    if not title:
+        return None
+    if not genre:
+        return None
+    if not rating:
+        return None
+    movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating        
+    }
+    return movie
+    
+def add_to_watched(user_data, movie):
+    if not movie:
+        return None
+    
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    if not movie:
+        return None
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    print("a")
+    for i in range(len(user_data["watchlist"])):
+        if title in user_data["watchlist"][i]["title"]:
+            user_data["watched"].append(user_data["watchlist"][i])
+            user_data["watchlist"].remove(user_data["watchlist"][i])
+
+    return user_data
+
+def get_watched_avg_rating(user_data):
+    total = 0
+    for each in range(len(user_data["watched"])):
+        total += user_data["watched"][each]["rating"]
```

Comment:
```suggestion
    for movie in user_data["watched"]:
        total += movie["rating"]
```
You can also iterate through the list itself rather than the indices. A slight change that makes your code a little less wordy.